.github/workflows/unittest.yml

```diff
@@ -54,21 +54,55 @@ jobs:
               - conda-forge
               - mcvine
               - mantid
-      - name: install multiphonon
-        run: |
-              python -m pip install -e .
-      - name: building python wheel
+      - name: Build python wheel
         shell: bash -l {0}
         run: |
           python -m build --wheel --no-isolation
+          check-wheel-contents dist/multiphonon-*.whl
+      - name: Upload python wheel
```

Comment:
Can you add a comment about this uploading as an artifact to the source repository rather than pypi/conda/whatever. It took me a bit to realize that and it'd be good to prevent misunderstandings.